run.sh

```diff
@@ -65,14 +65,14 @@ done
 GALAXY_VIRTUAL_ENV="${GALAXY_VIRTUAL_ENV:-.venv}"
 if [ -d "$GALAXY_VIRTUAL_ENV" -a -z "$skip_venv" ];
 then
-    [ -n "$PYTHONPATH" ] && { echo 'Unsetting $PYTHONPATH'; unset PYTHONPATH; }
```

Comment:
A 
```
# shellcheck disable=SC2016
```
line needs to be added before each line, or just at the start of the file if using the latest v0.4.4 release.